compiler/damlc/tests/daml-test-files/KindChecking.daml

```diff
@@ -0,0 +1,9 @@
+-- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+-- Make sure that invalid kinds are reported as errors by the DAML compiler.
+module KindChecking where
+
+-- @ERROR range=9:1-9:4; Nat kind on the right-hand side of kind arrow
```

Comment:
I suppose that's the error location you're referring to. I don't think it's a problem that this is slightly off since nobody should actually hit this.